Makefile

```diff
@@ -56,13 +56,12 @@ gen: ${GENALL}
 build-self: ensure-docker
 	(cd libs; cargo build --all)
 	(cd libs/gl-client-py && python3 -m maturin develop)
+	pip install -e libs/gl-testing
 
 check-self: ensure-docker
 	PYTHONPATH=/repo/libs/gl-testing \
 	pytest -vvv \
 	  /repo/libs/gl-testing \
```

Comment:
Is there any specific reason why we shall remove the amount processes? Is this docker related?

Reply:
I just don't like it too much on CI, because it actually slows CI down, and it has a tendency of messing up the stdout-capturing in pytest, causing a wall of text obscuring what we're actually looking at. CI is slow, but we don't really care too much, hence why running sequentially can be useful.

For my own testing I have a `pytest.ini` file in the repo root that has exactly the options I like when running tests locally.